client/main.css

```diff
@@ -14,6 +14,10 @@
   --success-green: #99cc33
 }
 
+.clean_link {
```

Comment:
You don't need to add this class for links. There's already a class called `.link` used to remove the default style from *a* tags.
Also, when creating new classes, please follow the naming convention that uses "snake casing" for multiple words (hyphens)

Reply:
(Seb here)

- kebab-case
- snake_case

Reply:
Good point :) Thanks for the correction